src/Wrapper.js

```diff
@@ -276,6 +280,19 @@ class Wrapper extends React.PureComponent {
               ],
             }))
           }
+          onMsgSignSuccess={({ data, description, identifier }) =>
```

Comment:
Review where these notifications are being used, and how we should format signature notifications (or if we even need them?)